fuzz/generate-files.sh

```diff
@@ -40,16 +40,16 @@ EOF
 done
 
 # 2. Generate .github/workflows/fuzz.yml
-cat > "$REPO_DIR/.github/workflows/fuzz.yml" <<EOF
+cat > "$REPO_DIR/.github/workflows/cron-daily-fuzz.yml" <<EOF
```

Comment:
This was totally broken no? The `fuzz.yml` doesn't exist on `master`

Reply:
Yep. Though the rename (and other changes) were fairly recent, so it's only been broken for a short time.

In any case, we should verify consistency in CI. But we can do that later after we've moved more stuff into rust-bitcoin-maintainer-tools, since it affects other repos beyond this one.

Reply:
It was only broken in the sense that we forget to update the `generate-files.sh` file and use it and always just hack the yaml file directly. The yaml file was/is correct.